src/microbe_stage/FloatingChunk.cs

```diff
@@ -143,7 +150,10 @@ public ChunkConfiguration CreateChunkConfigurationFromThis()
         config.Meshes = new List<ChunkConfiguration.ChunkScene>();
 
         var item = new ChunkConfiguration.ChunkScene
-            { LoadedScene = GraphicsScene, ScenePath = GraphicsScene.ResourcePath, SceneModelPath = ModelNodePath };
+        {
+            LoadedScene = GraphicsScene, ScenePath = GraphicsScene.ResourcePath, SceneModelPath = ModelNodePath,
+            LoadedCollisionShape = PhysicsMesh, CollisionShapePath = PhysicsMesh.ResourcePath,
```

Comment:
What if PhysicsMesh is null here?

Reply:
yeah, it should be `CollisionShapePath = PhysicsMesh?.ResourcePath`

Reply:
I didn't suggest that immediately obvious thing, as new chunks are spawned based on the data saved here, so there might be some extra work needed there to make that play nice with CollisionShapePath being null...

Reply:
I've looked into `CollisionShapePath`'s usages and there seems no trouble if it is null, but I might have missed something

Reply:
I do realize I need to assign `PhysicsMesh` the sphere shape here if it's null: https://github.com/Revolutionary-Games/Thrive/blob/1331c721a9b133df527081639e2b067604e511a9/src/microbe_stage/FloatingChunk.cs#L344 For the `CollisionShapePath` shouldn't it also work the same as `SceneModelPath`? Since it's null/empty(?) sometimes.

Reply:
But you have `CollisionShapePath = PhysicsMesh.ResourcePath` which will crash if PhysicsMesh is null